.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
If we're changing the format of this file, I'd rather it be changed to YAML. the JSON format for overrides is imo not much prettier than TOML